common4j/src/main/com/microsoft/identity/common/java/logging/DiagnosticContext.java

```diff
@@ -70,6 +70,11 @@ public IRequestContext getRequestContext() {
         return REQUEST_CONTEXT_THREAD_LOCAL.get();
     }
 
+    public String getThreadCorrelationId() {
+        return getRequestContext().containsKey(DiagnosticContext.CORRELATION_ID)
+                ? getRequestContext().get(DiagnosticContext.CORRELATION_ID) :"UNSET";
```

Comment:
Can we leverage an existing constant for UNSET? Or create one if it doesn't exist?